app/javascript/components/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, applyMiddleware, combineReducers } from "redux";
```

Comment:
- [ ] I had issues setting up your project because you are missing the package "redux" in your `package.json` file. I fixed it after running the following command; kindly do the same:

  ```
  npm i redux
  ```
  
  ![aea](https://user-images.githubusercontent.com/81468675/199082794-ce7113af-4e8a-4b90-af49-3d4596eecf10.jpg)